sqldelight-compiler/build.gradle

```diff
@@ -16,12 +16,12 @@ generateGrammarSource {
 
 dependencies {
   antlr dep.antlr
-  compile dep.antlrRuntime
-  compile dep.javaPoet
-  compile dep.kotlinStdLib
+  implementation dep.antlrRuntime
+  implementation dep.javaPoet
+  implementation dep.kotlinStdLib
```

Comment:
Yep. And since this artifact isn't really meant to be consumed by anyone except our own two plugins it's not a big deal that we don't have good encapsulation.